benchmarks/tracing_trace.rb

```diff
@@ -0,0 +1,48 @@
+# Used to quickly run benchmark under RSpec as part of the usual test suite, to validate it didn't bitrot
+VALIDATE_BENCHMARK_MODE = ENV['VALIDATE_BENCHMARK'] == 'true'
+
+return unless __FILE__ == $PROGRAM_NAME || VALIDATE_BENCHMARK_MODE
+
+require 'benchmark/ips'
+
+# Load the local version of the gem
+lib = File.expand_path('lib', __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
```

Comment:
Minor: I'm not sure this is needed if you run with `bundle exec`. And running without `bundle exec` is not a great since it may pull random versions of stuff, so I'm not sure it's worth supporting (?)

Reply:
Good suggestion, done!